recipe/meta.yaml

```diff
@@ -325,6 +332,18 @@ outputs:
         - ${CXX} -Wall tests/hello_world.cpp -c -o hello_world.o --std=c++17
         - ${CXX} -Wall hello_world.o -o hello_world --std=c++17
 
+        # test tzdb integration
+      {% if gcc_maj_ver|int >= 14 %}
+        - ${CXX} -isystem ${PREFIX}/include --std=c++20 -Wall tests/tzdb.cpp -c -o tzdb.o
+        {% if target_platform == cross_target_platform %}
+        - ${CXX} tzdb.o -o tzdb && ./tzdb
+        - strace ./tzdb 2>&1 | grep ${CONDA_PREFIX}/lib/../share/zoneinfo/tzdata.zi
```

Comment:
What do you think about this test? This should be enough IMO

Reply:
Fine with me. 👍